src/main/scala/net/snowflake/spark/snowflake/SnowflakeRelation.scala

```diff
@@ -201,6 +201,112 @@ private[snowflake] case class SnowflakeRelation(
     )
   }
 
+  // Print result set statistic information
+  private def printStatForSnowflakeResultSetRDD(
+     resultSetSerializables: Array[SnowflakeResultSetSerializable]) : Unit = {
+    var totalRowCount : Long = 0
+    var totalCompressedSize : Long = 0
+    var totalUnCompressedSize : Long = 0
+    var totalURLCount : Long = 0
+
+    resultSetSerializables.zipWithIndex.map {
```

Comment:
It seems like you don't need index from here. and if you only want to scan the collection without return value, you can use `foreach` instead of `map`

Reply:
I will update it to use FOREACH.